<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticket Dashboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/res/images/BitTicketLogo.png</normaloff>:/new/res/images/BitTicketLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Review</string>
    </property>
    <widget class="QLabel" name="lb_rating">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_rating">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>30</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_total_closed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Incident Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_total_solved">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ticket Status </string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_Incstatus">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>60</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_Tickstatus">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>90</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_remove">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>REMOVE TICKET</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(2, 173, 237);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>TICKET DASHBOARD</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_tickets">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>251</width>
      <height>341</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of Tickets&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_modify">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>560</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MODIFY TICKET</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Ticket Details</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Symptoms</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_level_support">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>233</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Level of Support</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_ticket_num">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>129</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Ticket Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_timestamp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>204</y>
       <width>129</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Timestamp</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>261</y>
       <width>129</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Agent</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_priority">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>176</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Prioity</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_impact">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>119</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Impact</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_urgency">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>147</y>
       <width>129</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Urgency</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_incident_cat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>129</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Incident Category</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tags">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>129</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_ticketnum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ticket Number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_symptoms">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>291</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_Agent">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>262</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_timestamp">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>204</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_levelsupport">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>233</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_Incidentcat">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>59</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_impact">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>117</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_priority">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>175</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_tags">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>88</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_urgency">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>146</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Contact Details</string>
    </property>
    <widget class="QLabel" name="lb_display_phone">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_email">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>61</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_display_name">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>32</y>
       <width>179</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_phone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>59</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_email">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>59</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>33</y>
       <width>59</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/res/images/BitTicketLogo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage">
    <property name="geometry">
     <rect>
      <x>669</x>
      <y>270</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionModify_Ticket"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionSpyBot"/>
     <addaction name="actionGravira"/>
    </widget>
    <addaction name="menuThemes"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New Ticket</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load Tickets</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Tickets</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionModify_Ticket">
   <property name="text">
    <string>Modify Ticket</string>
   </property>
  </action>
  <action name="actionSpyBot">
   <property name="text">
    <string>Darkmode</string>
   </property>
  </action>
  <action name="actionGravira">
   <property name="text">
    <string>Lightmode</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lst_tickets</tabstop>
  <tabstop>pb_remove</tabstop>
  <tabstop>pb_modify</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
